content/en/account_management/billing/_index.md

```diff
@@ -33,43 +33,53 @@ Datadog meters the count of IoT devices hourly. The billable count of IoT device
+You can view your billing contact details on the [**Billing Contact Details**][8] section. 
 
-The [**Payment Method**][12] section contains details on your payment options. 
+{{< img src="account_management/billing/BillingContactDetailsOverview.png" alt="Billing contact details on the Plan page" style="width:90%;" >}}
 
-{{< img src="account_management/billing/PaymentMethodOverview.png" alt="Payment method on the Plan page" style="width:80%;" >}}
+Through **Edit Details**, you can add, edit or remove your billing address. You can also set specific email addresses to receive invoices.
 
-**Edit Payment** provides options to manage payment methods. You can edit or remove cards, and request to change your payment method from card to invoice and vice versa. 
+{{< img src="account_management/billing/BillingContactDetailsEdit.png" alt="Editing billing contact details on the Plan page" style="width:90%;" >}}
 
-{{< img src="account_management/billing/PaymentSettingsDetails.png" alt="Payment settings on the Plan page" style="width:80%;" >}}
+**Note**: The email address does not need to be a team member within Datadog. For example, you could use `invoices@yourcompany.com`.
 
-### Managing your billing contact details
+### View your subscription details
 
-You can view your billing contact details on the [**Billing Contact Details**][12] section. 
+The [Subscription Details][8] section includes the quantity, contract price and on-demand price for all committed products.
 
-{{< img src="account_management/billing/BillingContactDetailsOverview.png" alt="Billing contact details on the Plan page" style="width:80%;" >}}
+{{< img src="/account_management/billing/subscription_details.png" alt="Account Plan & Usage page highlighting Subscription Details section" style="width:90%;" >}}
 
-Through **Edit Details**, you can add, edit or remove your billing address. You can also set specific email addresses to receive invoices.
+**Note**: If your billing is managed directly through a Datadog Partner, Subscription Details are not supported.
 
-{{< img src="account_management/billing/BillingContactDetailsEdit.png" alt="Editing billing contact details on the Plan page" style="width:80%;" >}}
+## Payment
 
-**Note**: The email address does not need to be a team member within Datadog. For example, you could use `invoices@yourcompany.com`.
+There are two choices for payment method:
+- Credit card
+- Invoicing (ACH, wire, or check)
+
+### Credit card
+
+If you pay by credit card, receipts are available to [Administrators][9] for previous months under [Billing History][10]. For copies of your invoice, email [Datadog billing][11].
+
+See [Credit Card Billing][12] for more details.
+
+### Invoicing
+
+If you pay by check, ACH, or wire, invoices are emailed to the billing email addresses near the 10th business day of each month. If you need an additional copy, email [Datadog billing][11]. Details on where to remit payment can be found on the invoice.
```

Comment:
```suggestion
If you pay by check, ACH, or wire, invoices are emailed to the billing email addresses near the 10th business day of each month. To request an additional copy, email [Datadog billing][11]. Details on where to remit payment can be found on the invoice.
```

Reply:
Re: where we say "near the 10th business day of each month..." I'm wondering if there's a way for us to be more specific. Do you happen to know how this works? Is there a date range we can specify here?

Reply:
No, I'm not entirely sure on this, but I could ask!